public/data/unseoJang/commentData.json

```
@@ -0,0 +1,22 @@
+[
+    {
+        "key" : 1,
+        "userId": "kimdaebeom",
+        "comment": "천안와서 돌아이가 연락을 안했넹?😮1"
+    },
+    {
+        "key": 2,
+        "userId": "kimdaebeom",
+        "comment": "천안와서 돌아이가 연락을 안했넹?😮2"
+    },
+    {
+        "key": 3,
+        "userId": "kimdaebeom",
+        "comment": "천안와서 돌아이가 연락을 안했넹?😮3"
+    },
+    {
+        "key": 4,
+        "userId": "kimdaebeom",
+        "comment": "천안와서 돌아이가 연락을 안했넹?😮4"
```

누군진 모르겠지만 잘못했네요 연락안하신분이

---

죄송합니다...제가 잘못햇네요 ㅋㅋㅋ
```suggestion
        "comment": "천안와서 돌아이가 연락을 안했넹?😮4"
```